exercises/anagram/.meta/hints.md

```diff
@@ -0,0 +1,5 @@
+# Hints
+
+The solution is case insensitive, that means `"WOrd"` is the same with `"word"` or `"woRd"`. It may help to take a peak in the [std library](https://doc.rust-lang.org/std/index.html) for functions that can convert between them.
```

Comment:
I think that if we include a link, it should be to a more useful page than the docs homepage. Suggest https://doc.rust-lang.org/std/primitive.str.html. I would also accept removing the link here.

Reply:
from this link, the solution is one search away. I thing its good experience searching for what you want. Maybe I should add "search for char" or something similar